### ISSUE TITLE
Cannot upload OVA template if eula format is not supported

### ISSUE THREAD

**Issue by weizhouapache** [apache/cloudstack#8039]
If OVF has the following eula, the template upload will fail

```
<ovf:EulaSection><ovf:Info>An end-user license agreement</ovf:Info><ovf:License>
…. (license text)
</ovf:License></ovf:EulaSection>
```

what ACS supports is
```
<EulaSection><Info>An end-user license agreement</Info><License>
…. (license text)
</License></EulaSection>
```

However,the first format is supported by vmware, ACS should support it as well.

##### ISSUE TYPE

 * Bug Report

##### COMPONENT NAME

~~~

~~~

##### CLOUDSTACK VERSION

~~~

~~~

##### STEPS TO REPRODUCE

~~~

~~~

##### EXPECTED RESULTS

~~~

~~~

##### ACTUAL RESULTS

~~~

~~~

**Comment by harikrishna-patnala** [comment]
@weizhouapache I think template registration also fails, am I right! or is it just upload template operation ?

**Comment by weizhouapache** [comment]
> @weizhouapache I think template registration also fails, am I right! or is it just upload template operation ?

@harikrishna-patnala 
if eula is checked during template registration (read properties from OVF), it will fail as well.
I have not tested it yet.

**Comment by shwstppr** [comment]
Fixed by #8081
